apps/chatbot/src/modules/chatbot.py

```diff
@@ -127,10 +127,13 @@ def _unmask_reference(self, response_str: str, nodes) -> str:
     
 
     def mask_pii(self, message: str) -> str:
-        try:
-            return self.pii.mask_pii(message)
-        except Exception as e:
-            logging.warning(f"[chatbot.py - mask_pii] exception in mask_pii: {e}")
+        if USE_PRESIDIO:
```

Comment:
is this env var still necessary or can we remove it?

Reply:
for now we keep it